docs/embedded/v8/pre-configuring-emails.md

```diff
@@ -59,4 +59,24 @@ try {
 }
 ```
 
+When using `throw_on_cancel` all errors resulting in not sending an email throw an exception:
+
+```php
+<?php
+
+declare(strict_types=1);
+
+try {
+    $result = $mailService->send('welcome', [
+        'to' => ['new-user@gmail.com'
+    ]]);
+
+    // Email properly sent
+} catch (AcMailer\Exception\MailCancelledException $e) {
+    // Email was cancelled
+} catch (AcMailer\Exception\MailException $e) {
+    // Error sending email
+}
+```
+
```

Comment:
Let's keep this here for now, as it's the section where the error handling is currently explained.

However, the whole error handling should probably be in "Usage". Let's tackle that on a separated PR after this one is merged.